tests/test_opds.py

```diff
@@ -1282,20 +1303,71 @@ def make_page():
 
         # The new work does not show up in the feed because
         # we get the old cached version.
-        feed2 = make_page()
-        assert work2.title not in feed2
+        response2 = make_page()
+        assert work2.title not in unicode(response2)
         assert cached.timestamp == old_timestamp
 
         # Change the WorkList's MAX_CACHE_AGE to disable caching, and
         # we get a brand new page with the new work.
         fantasy_lane.MAX_CACHE_AGE = 0
-        feed3 = make_page()
+        response3 = make_page()
         assert cached.timestamp > old_timestamp
-        assert work2.title in feed3
+        assert work2.title in unicode(response3)
 
 
 class TestAcquisitionFeed(DatabaseTest):
 
+    def test_page(self):
+        # Verify that AcquisitionFeed.page() returns an appropriate OPDSFeedResponse
+
+        wl = WorkList()
+        wl.initialize(self._default_library)
+        private = object()
+        response = AcquisitionFeed.page(
+            self._db, "feed title", "url", wl, TestAnnotator,
+            max_age=10, private=private
+        )
+
+        # The result is an OPDSFeedResponse. The 'private' argument,
+        # unused by page(), was passed along into the constructor.
+        assert isinstance(response, OPDSFeedResponse)
+        eq_(10, response.max_age)
+        eq_(private, response.private)
+
+        assert '<title>feed title</title>' in response.data
+
+    def as_response(self):
+        # Verify the ability to convert an AcquisitionFeed object to an
+        # OPDSFeedResponse containing the feed.
+        feed = AcquisitionFeed(self._db, "feed title", [], TestAnnotator)
+        response = feed.as_response(max_age=101)
+        eq_(200, response.status)
+        assert isinstance(response, OPDSFeedResponse)
+        assert '<title>feed title</title>' in response.data
+        eq_(101, response.max_age)
+
+    def as_response(self):
```

Comment:
Are these two method names correct? They're the same and don't start with `test_`.

Reply:
Yeah, these tests weren't running at all.